assign-2/src/data/CreateStudentMenuItem.java

```diff
@@ -0,0 +1,32 @@
+package data;
+
+import data.MenuItem;
+import data.StudentDao;
+import logic.Student;
+
+import java.util.Scanner;
+
+public class CreateStudentMenuItem extends MenuItem {
+    private final StudentDao studentDao;
```

Comment:
Very good choice in choosing the access modifier `final` and also making it `final`.